### ISSUE TITLE
[BUG] Still getting lua error with sub rogue

### ISSUE THREAD

**Issue by wheatbread** [Hekili/hekili#2497]
### Before You Begin

- [ ] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### Describe the Issue

Sorry, I don't know if the current version is supposed to be the fixed one, but in case it is, still getting spammed with the same error from here:  https://github.com/Hekili/hekili/issues/2483

`27x Hekili/Dragonflight/RogueSubtlety.lua:422: bad argument #1 to 'wipe' (table expected, got nil) [string "=[C]"]: ? [string "@Hekili/Dragonflight/RogueSubtlety.lua"]:422: in function hook'
[string "@Hekili/Classes.lua"]:5728: in function callHook' [string "@Hekili/Events.lua"]:1557: in function <Hekili/Events.lua:1555> [string "@Hekili/Events.lua"]:1852: in function handler'
[string "@Hekili/Events.lua"]:60: in function <Hekili/Events.lua:52>

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"`

### How to Reproduce

1. Be sub
2. Enter combat

### Player Information (Link)

https://pastebin.com/n5hrX1r1

### Error Messages (Link)

https://pastebin.com/0GXJAJHq

**Comment by Hekili** [comment]
Your error messages are several versions old.  Make sure your addon manager is installing in the right game folder.  From your Player Information:

> build: **v10.0.7-1.0.0a**
> level: 70 (70)
> class: ROGUE
> spec: subtlety

The current build would be v10.0.7-1.0.3.

**Comment by Hekili** [comment]
Your error messages are several versions old.  Make sure your addon manager is installing in the right game folder.  From your Player Information:

> build: **v10.0.7-1.0.0a**
> level: 70 (70)
> class: ROGUE
> spec: subtlety

The current build would be v10.0.7-1.0.3.

**Comment by wheatbread** [comment]
Hi,

Sorry about that, I had copied the links from the previous ticket thinking it would be the same. 

I uninstalled Hekili to make sure, and via Wowup reinstalled it now with v10.0.7-1.0.4, released 22 hours ago as I write.  

I then went to the target dummies and dpsed them as sub.  Sadly got errors. :( 

Here are new pastebin links:

Lua error: https://pastebin.com/FWhb3AQN
Issue Reporting: https://pastebin.com/PjqfNWqk
